django_lightweight_queue/job.py

```diff
@@ -1,21 +1,38 @@
 import sys
 import json
 import time
+import datetime
 
 from .utils import get_path, get_middleware
 
 class Job(object):
-    def __init__(self, path, args, kwargs, timeout=None, sigkill_on_stop=False):
+    def __init__(
+        self,
+        path,
+        args,
+        kwargs,
+        timeout=None,
+        sigkill_on_stop=False,
+        created_time=None,
+    ):
         self.path = path
         self.args = args
         self.kwargs = kwargs
         self.timeout = timeout
         self.sigkill_on_stop = sigkill_on_stop
+        self.created_time = (
+            created_time or datetime.datetime.utcnow().isoformat(sep=' ')
```

Comment:
1. Do we ever want to support passing in a creation time? I.e. can we just default to `datetime.datetime.utcnow()`.?
2. I think it'd be less surprising to have `self.created_time` stay as a datetime instance until we need to stringify it.